### ISSUE TITLE
app: Web gl warning causing performance reduction

### ISSUE THREAD

**Issue by alisaad673** [webaverse/app#2661]
These warnings appear randomly at any time during gameplay. Most probably it appears right after switching scenes. This causes extreme performance reduction. FPS reduce to almost less than half after some time. Fps keep reducing with time.

![wbgl](https://user-images.githubusercontent.com/35926530/160156559-af2c8f8c-ac27-4bf9-aeef-a1e1f86a81ae.PNG)
![webgl](https://user-images.githubusercontent.com/35926530/160156565-d3f90518-5ea9-4044-980c-d7f6b43e6c3b.PNG)

**Comment by avaer** [comment]
Sounds like a bug with switching render settings, or some app is not behaving nicely on scene switch.

This needs a consistent reproduction.

**Comment by avaer** [comment]
Note that scene switching is not supported until the apps inside the scenes are cleaning up after themselves. Does this only happen when switching scenes?

**Comment by avaer** [comment]
This  is likely tied to a specific object as well, so it would be good to know which object it is.

**Comment by avaer** [comment]
Invalid query id is related to performance tracking.

**Comment by avaer** [comment]
This is likely to occur when there are two apps with the same name in the scene, since that is used as a discriminator to check the currently queried app.

I think it would be better to base it on the `module` that the app loaded. That would probably solve this bug.

**Comment by avaer** [comment]
See here: https://github.com/webaverse/app/blob/d683c461cbc6eac6318b41cff5e3b83b57cd7c07/performance-tracker.js#L69

**Comment by avaer** [comment]
I merged a potential fix. Can you test again?

**Comment by alisaad673** [comment]
I have tested it again. The issue is not happening with scene switch anymore. But in silkfountains scn, it appears after 1-3 minutes of gameplay. 
-It only happens when "H" stats are on.
-Tested in all scenes, but right now it appeared only in silk fountains.
-Unlike before, it happens now only after 1-3 minutes of game play, while h stats are on.

**Comment by avaer** [comment]
Thanks for the info. This probably means that silk fountains has two instances of apps with the same exact modules being rendered, causing hashes to collide.

We might need a better approach towards tracking uniqueness of apps in this case; perhaps we should tack on the app id.

**Comment by alisaad673** [comment]
Closing in favor of #2663
